homeassistant/components/aemet/sensor.py

```diff
@@ -51,172 +83,262 @@
     ATTR_API_WIND_MAX_SPEED,
     ATTR_API_WIND_SPEED,
     ATTRIBUTION,
+    CONDITIONS_MAP,
     DOMAIN,
     ENTRY_NAME,
     ENTRY_WEATHER_COORDINATOR,
-    FORECAST_MODE_ATTR_API,
-    FORECAST_MODE_DAILY,
-    FORECAST_MODES,
-    FORECAST_MONITORED_CONDITIONS,
-    MONITORED_CONDITIONS,
 )
+from .entity import AemetEntity
 from .weather_update_coordinator import WeatherUpdateCoordinator
 
-FORECAST_SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
-    SensorEntityDescription(
-        key=ATTR_API_FORECAST_CONDITION,
-        name="Condition",
+
+@dataclass
+class AemetSensorEntityDescription(SensorEntityDescription):
+    """A class that describes AEMET OpenData sensor entities."""
+
+    keys: list[str] | None = None
+
+
+FORECAST_SENSORS: Final[tuple[AemetSensorEntityDescription, ...]] = (
+    AemetSensorEntityDescription(
+        key=f"forecast-daily-{ATTR_API_FORECAST_CONDITION}",
+        keys=[AOD_TOWN, AOD_FORECAST_DAILY, AOD_FORECAST_CURRENT, AOD_CONDITION],
+        name="Daily Forecast Condition",
+    ),
+    AemetSensorEntityDescription(
+        entity_registry_enabled_default=False,
+        key=f"forecast-hourly-{ATTR_API_FORECAST_CONDITION}",
+        keys=[AOD_TOWN, AOD_FORECAST_HOURLY, AOD_FORECAST_CURRENT, AOD_CONDITION],
+        name="Hourly Forecast Condition",
```

Comment:
```suggestion
        name="Hourly Forecast Condition",
        value_fn: lambda value: CONDITIONS_MAP.get(value)
```

Reply:
Done in f064f9e010966a5c9832823cc324e04c7e348d2a